Gemfile

```diff
@@ -2,4 +2,4 @@ source 'https://rubygems.org'
 
 gem 'jekyll'
 gem 'jekyll-redirect-from'
-gem 'rouge'
+gem 'rouge', :git => 'https://github.com/kylef/rouge', :branch => 'kylef/apiblueprint-lexer'
```

Comment:
I certainly can, I didn't want to create additional noise with extra repos (and hoping this would be short term and that my PR gets merged into the Rouge gem). I will move this over in the morning.